lib/ui/page/home/page/chat/forward/controller.dart

```diff
@@ -68,7 +68,7 @@ class ChatForwardController extends GetxController {
 
   /// Callback, called when a [ChatForwardView] this controller is bound to
   /// should be popped from the [Navigator].
-  final void Function()? pop;
+  final void Function([dynamic])? pop;
```

Comment:
@SleepySquash Нельзя тут сделать так?
```suggestion
  final void Function([bool])? pop;
```